src/modules/tableValues.js

```diff
@@ -39,6 +41,17 @@ const getFormattedCellValue = ({ value, header = {}, visualization = {} }) => {
         return getStatusNames()[value] || value
     }
 
+    // TODO: Remove this when the problem with empty/partial user info is resolved in backend
```

Comment:
This comment implies there is an open Jira issue for the empty/partial user info. Could you include the Jira issue number in the comment?